src/services/services.ts

```diff
@@ -1958,6 +1958,12 @@ namespace ts {
                     (char >= CharacterCodes.A && char <= CharacterCodes.Z) ||
                     (char >= CharacterCodes._0 && char <= CharacterCodes._9);
             }
+
+            function isNodeModulesFile(path: string): boolean {
+                const node_modulesFolderName = "/node_modules/";
+
+                return path.indexOf(node_modulesFolderName) !== -1;

```

Comment:
My favorite is:

```javascript
  return !!~path.indexOf(somevar); // Note: Overly cute hack. Do not use! :-)
```

Take your time.... I'll wait :wink:

Note in JavaScript it is explicitly a result of -1 if not found (http://es5.github.io/#x15.5.4.7). Simply checking for < 0 seems less correct, in the unlikely event they add other negative values for other scenarios.